notebooks/json_blocks/WET/BLOCK-T419.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        [1, 1, 1, 1, 0.5],
</pre><p>Adding these ranges to the Zephyr scale to have them handy might be helpful. Not as important here as for WET-001 though I don't think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>